charts/linkerd2/templates/proxy-injector-rbac.yaml

```diff
@@ -83,11 +83,15 @@ metadata:
 webhooks:
 - name: linkerd-proxy-injector.linkerd.io
   namespaceSelector:
+    {{- if .Values.proxyInjector.namespaceSelector }}
```

Comment:
Why not just set the default to the current value? That way there isn't an if/else statement.

Reply:
The original version did, but it caused the tests to fail :sweat_smile:

Reply:
Shouldn't this also have kube-system here by default?

Reply:
> Happy to change it back as it would be my preferred solution, but it may need changes beyond just to the helm chart

I think I would prefer the default values approach. Additional changes are fine.